### DISCUSSION TITLE
Welcome to mpesa-stk-wc Discussions!

### DISCUSSION THREAD

**Discussion by mwamodo** [apxcde/mpesa-stk-wc#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BradKada** [comment]
Hi, I'm getting an error message = Could not connect to MPesa to process payment.
what could be an issue?

**Reply by mwamodo** [reply]
Didn't see this on time, did you get it resolved?

**Comment by dickienertwork** [comment]
how do i integrate mpesa to my codeigniter project?

**Reply by mwamodo** [reply]
This is a wordpress plugin, you cannot add this to a codeigniter project
